starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -6,34 +6,53 @@
 }
 
 h1 {
-    text-align: center;
+    font-size: 65px;
+    position: absolute;
+    top: 150px;
+    left: 38%;
+    color: white;
+    font-weight: 300;
 }
 
-header {
-    /* background-image: url("images/apple-pie.jpg"); */
-    /* text-align: center; */
-    /* background-size: cover; */
-    background-image: url("images/apple-pie.jpg");
-    background-position: 0 0;
-    background-repeat: no-repeat;
-    background-size: cover;
-    height: 650px;
+.cover-photo {
+    width:80%;
+    height: 500px;
+    margin: auto;
+    overflow: scroll;
```

Comment:
```
overflow: hidden;
```
And that will fix your issue with scroll bar.

Reply:
I had overflow:hidden before but it still gives issues. check the new pull request I just sent with the changes.

Reply:
I don't see any issues when using hidden. What is the issue you're dealing with?